hooks/useVercelChat.ts

```diff
@@ -50,6 +56,7 @@ export function useVercelChat({ id, initialMessages }: UseVercelChatProps) {
       accountId: userId,
       email: userData?.email,
     },
+    // initialMessages,
```

Comment:
this should have been removed from `main`
are you sure you have the latest changes from `main`?
`git pull origin main`
- overall: delete unused code.